department-of-veterans-affairs/caseflow
# Allow bva admins to modify organizations

Resolves #13748

### Description
Allows BVA team members to view and modify organization members

### Acceptance Criteria
- [x] BVA team members can view organization team management pages
- [x] BVA team members can add users to orgs
- [x] BVA team members can remove users from orgs
- [x] BVA team members can modify admin rights of users in orgs

### Testing Plan
1. Create a new BVA user that is not a system admin
```ruby
user = User.create!(station_id: 101, css_id: "BVA_ADMIN", full_name: "John BvaAdmin Bosworth")
Bva.singleton.add_user(user)
```
2. Sign in as this user and go to an org page! http://localhost:3000/organizations/bva/users
3. Verify no errors
4. Add a team member and verify no errors
5. Remove a team member and verify no errors
6. Add admin right to team member and verify no errors
7. Remove admin rights from team member and verify no errors

### User Facing Changes
BEFORE|AFTER
---|---
![Screen Shot 2020-04-21 at 5 56 37 PM](https://user-images.githubusercontent.com/45575454/79918634-d3ff2d80-83fa-11ea-8901-1cce77a7e533.png)|![Screen Shot 2020-04-21 at 5 57 39 PM](https://user-images.githubusercontent.com/45575454/79918642-d792b480-83fa-11ea-81ae-e1c78c041ab1.png)
